File: Graphs/printAdjacencyList.java
import java.util.ArrayList;
import java.util.List;

// User function Template for Java
class Solution {
    public List<List<Integer>> printGraph(int V, int edges[][]) {
        List<List<Integer>> adjList = new ArrayList<>();
        for(int i = 0; i < V; i++)
            adjList.add(new ArrayList<>());
        for(int i = 0; i < edges.length; i++) {
            adjList.get(edges[i][0]).add(edges[i][1]);
            adjList.get(edges[i][1]).add(edges[i][0]);
        }
        return adjList;
    }
}

File: Graphs/Print_adjacency_list.cpp
// https://www.geeksforgeeks.org/problems/print-adjacency-list-1587115620/1

#include<bits/stdc++.h>
using namespace std;

class Solution {
  public:
    vector<vector<int>> printGraph(int V, vector<pair<int, int>>& edges) {
        vector<vector<int>> adj(V);
        for(auto& e : edges) {
            adj[e.first].push_back(e.second);
            adj[e.second].push_back(e.first);
        }
        return adj;
    }
};